File: core/src/main/java/com/ustadmobile/core/view/Register2View.java
package com.ustadmobile.core.view;

public interface Register2View extends UstadView {

    String VIEW_NAME = "RegisterAccount";

    int FIELD_FIRST_NAME = 1;

    int FIELD_LAST_NAME = 2;

    int FIELD_USERNAME = 3;

    int FIELD_EMAIL = 4;

    int FIELD_PASSWORD = 5;

    int FIELD_CONFIRM_PASSWORD = 6;

    void setErrorMessageView(String errorMessageView);

    void setServerUrl(String url);

    void setInProgress(boolean inProgress);

}

File: core/src/main/java/com/ustadmobile/core/view/Register2View.kt
package com.ustadmobile.core.view

interface Register2View : UstadView {

    fun setErrorMessageView(errorMessageView: String)

    fun setServerUrl(url: String)

    fun setInProgress(inProgress: Boolean)

    companion object {

        val VIEW_NAME = "RegisterAccount"

        val FIELD_FIRST_NAME = 1

        val FIELD_LAST_NAME = 2

        val FIELD_USERNAME = 3

        val FIELD_EMAIL = 4

        val FIELD_PASSWORD = 5

        val FIELD_CONFIRM_PASSWORD = 6
    }

}
